cognitive/src/main/scala/com/microsoft/ml/spark/cognitive/TextTranslator.scala

```diff
@@ -171,6 +211,8 @@ class Translate(override val uid: String) extends TextTranslatorBase(uid)
 
   def setToLanguage(v: Seq[String]): this.type = setScalarParam(toLanguage, v)
 
+  def setToLanguage(v: String): this.type = setScalarParam(toLanguage, Seq(v))
```

Comment:
Nice! We might need to add this to python methods too

Reply:
I guess this will automatically work? We're calling `self._java_obj = self._java_obj.setXXX(value)` and the java object will find the corresponding set function?

Reply:
Youre right!